crates/biome_html_analyze/tests/specs/a11y/useSemanticElements/html/invalid.html

```
@@ -0,0 +1,45 @@
+<!-- should generate diagnostics -->
+<div role="checkbox"></div>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)